src/SudokuSolver/SudokuSolverLib/SudokuPuzzle.cs

```diff
@@ -71,8 +71,38 @@ public static SudokuPuzzle Create(int boxWidth, int boxHeight, int hintsCount)
 
             return resultGrid;
         }
+		public static SudokuPuzzle Create(int boxWidth, int boxHeight, int hintsCount, int Seed)
```

Comment:
Since the code is identical (except for the seed value) I would replace the body of the original Create method with a call to this method where the seed is ((int)DateTime.Now.Ticks).

That way we avoid duplicating the code :).